README.md

```diff
@@ -76,6 +76,31 @@ The important files to check in each example are:
 The concepts in this package are demonstrated on the examples of *RRBot* and *DiffBot*.
 Those two world-known imaginary robots are trivial simulations to demonstrate and test `ros2_control` concepts.
 
+## Running
```

Comment:
Running of what?

this section is all about docker - so we should state this too

Reply:
Please let me know if this is better